### ISSUE TITLE
Unable to install on R 3.3.0 on Windows 10

### ISSUE THREAD

**Issue by hongyuanjia** [mkuhn/dict#4]
Hi @mkuhn . 

I noticed that the last commit `dict` package was about one year ago. And I failed to install it on my Windows machine complaining that the `dict` package is not available for R 3.3.3.

Would it be possible for you to update this package for newer version of R?

Thanks.

**Comment by mkuhn** [comment]
Did you follow the installation instructions on the readme?

```
if (!require("devtools")) install.packages("devtools")
devtools::install_github("mkuhn/dict")
```

**Comment by twolodzko** [comment]
@mkuhn will you consider posting it on CRAN? I'd potentially like to use your package as a dependency to package that I'm planning to implement.

**Comment by mkuhn** [comment]
No, I won't. While I'm happy that the package is useful for you, I don't
have the resources to maintain a CRAN package. Sorry! But this being open
source, feel free to adopt this package.
